### PR TITLE
Check for existing wrapper before creating it

### PR DESCRIPTION
**Pull request by cmaddalozzo**
Hey Ilya,

I've been using remodal on a Meteor project I've been working on. It's great! One issue that I was having was I needed to keep the original .remodal DOM element in place otherwise my template's event handlers wouldn't be able to keep track of it. An acceptable solution I found was to preemptively embed the .remodal inside of a .remodal-wrapper so that the plugin didn't need to create one, thus leaving the DOM tree untouched. The only issue I could foresee was undesired behaviour if somebody created a wrapped remodal deep in the DOM tree. Since pre-wrapping the remodal requires a concious effort by the dev and the default behaviour is unaffected I figure this shouldn't be cause for concern.

I've added a test to make sure everything works as expected. If this looks good to you I'd love to see this merged into master.

Thanks!
Curtis

### CODE REVIEW

**Comment by artshostak** [comment]
Would love to use this!

File: dist/remodal.js [hunk 1]
```diff
@@ -511,15 +511,23 @@
       namespacify('is-initialized') + ' ' +
```
Comment (VodkaBears) [inline dist/remodal.js]: Remove the dist folder form the commit, please.

File: test/remodal_test.js [hunk 2]
```diff
@@ -37,7 +37,13 @@
     var $elem = $('[data-remodal-id=modal3]');
 
     assert.ok($elem.remodal(), 'instance exists');
-    assert.equal($elem.data('remodal'), 2, 'index is right');
+  });
+
+  QUnit.test('Existing wrapper', function(assert) {
+    var $modal = $('[data-remodal-id=modal5]');
+    var $existingWrapper = $modal.parent();
+    var remodal = $modal.remodal();
```
Comment (VodkaBears) [inline test/remodal_test.js]: Add an empty line after this one, please.

File: src/remodal.js [hunk 3]
```diff
@@ -502,15 +502,23 @@
       namespacify('is-initialized') + ' ' +
       remodal.settings.modifier + ' ' +
       namespacify('is', STATES.CLOSED));
-
-    remodal.$wrapper = $('<div>')
-      .addClass(
-        namespacify('wrapper') + ' ' +
+    var $existingWrapper = remodal.$modal.parent('.' + namespacify('wrapper'));
```
Comment (VodkaBears) [inline src/remodal.js]: Move `var $existingWrapper;` to the top of the function after all declarations, please, and an empty lines before and after, please, please, please. :cake:

**Comment by VodkaBears** [comment]
Add, please, any note to the README.

I am ready to merge if there is no better solution.

**Comment by VodkaBears** [comment]
You removed the dist folder from the repo =). Do not not touch it at all.

File: src/remodal.js [hunk 4]
```diff
@@ -502,15 +503,23 @@
       namespacify('is-initialized') + ' ' +
       remodal.settings.modifier + ' ' +
       namespacify('is', STATES.CLOSED));
-
-    remodal.$wrapper = $('<div>')
-      .addClass(
-        namespacify('wrapper') + ' ' +
+    $existingWrapper = remodal.$modal.parent('.' + namespacify('wrapper'));
```
Comment (VodkaBears) [inline src/remodal.js]: Add empty lines before and after.

**Pull request closed without merging**